src/pages/Home.jsx

```diff
@@ -1,3 +1,64 @@
 import React from 'react';
+import { FormikSearch } from '../components/SearchForm.jsx';
 
-export default () => <div><h3>Hello world</h3></div>;
+export default class HomeLayout extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      searchTerms: '',
+      location: '',
+      radius: '',
+      timeframe: '',
+    }
+
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.handleSearchTerms = this.handleSearchTerms.bind(this)
+    this.handleLocation = this.handleLocation.bind(this)
+    this.handleRadius = this.handleRadius.bind(this);
+    this.handleTimeframe = this.handleTimeframe.bind(this)
+  }
+
+  handleSearchTerms(e) {
+    this.setState({searchTerms: e.target.value})
+  }
+
+  handleLocation(e) {
+    this.setState({location: e.target.value})
+  }
+
+  handleRadius(e) {
+    this.setState({radius: e.target.value})
+  }
+  handleTimeframe(e) {
+    this.setState({timeframe: e.target.value})
+  }
+
+  handleSubmit() {
+    event.preventDefault();
```

Comment:
Does this work?

I would assume you'd need to reference the `event` via the submit event argument.

```
handleSubmit(e) {
  e.preventDefault();
  ...
}
```